### PR TITLE
frontpage-redesign

### PR DESCRIPTION
**Pull request by laudickson**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -41,35 +41,11 @@
...
-                <img class="img" alt="Live webcam sex! More than 20000 Hot Girls are waiting for you!" src="//static1.awestatic.com/npt/banner/s1_girl/160xx600.jpg?sid=9300fc3b&psid=tightfitcams&pstool=201_2&psprogram=pps&campaign_id=" border="0" />
-              </a>
-
-              <!-- Exoclick Game Ad -->
-              <script type="text/javascript">var ad_idzone = "2808188", ad_width = "160", ad_height = "600";</script>
-              <script type="text/javascript" src="https://ads.exosrv.com/ads.js"></script>
-              <noscript><a href="https://main.exosrv.com/img-click.php?idzone=2808188" target="_blank"><img src="https://syndication.exosrv.com/ads-iframe-display.php?idzone=2808188&output=img&type=160x600" width="160" height="600"></a></noscript>
-              <!-- Exoclick Popunder -->
-              <script type="text/javascript">var ad_idzone = "2822626", ad_frequency_period = 720, ad_frequency_count = 1;</script>
-              <script type="text/javascript" src="https://ads.exosrv.com/popunder1000.js"></script>
```
Comment (jscho13) [inline app/views/layouts/application.html.erb:67]: wait we need this though

**Comment by jscho13** [comment]
the hover is sick

**Comment by laudickson** [comment]
oh shit i forgot to re-add the ads. putting back in. nice catch :+1:

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (laudickson) [reply app/views/layouts/application.html.erb:67]: totally my bad. adding back in all of them

File: app.css [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/Users/dickson/.rvm/gems/ruby-2.4.1/gems/sass-3.5.3/lib/sass/exec/sass_scss.rb:360:in `watch_or_update'
+/Users/dickson/.rvm/gems/ruby-2.4.1/gems/sass-3.5.3/lib/sass/exec/sass_scss.rb:51:in `process_result'
+/Users/dickson/.rvm/gems/ruby-2.4.1/gems/sass-3.5.3/lib/sass/exec/base.rb:52:in `parse'
+/Users/dickson/.rvm/gems/ruby-2.4.1/gems/sass-3.5.3/lib/sass/exec/base.rb:19:in `parse!'
+/Users/dickson/.rvm/gems/ruby-2.4.1/gems/sass-3.5.3/bin/sass:13:in `<top (required)>'
+/Users/dickson/.rvm/gems/ruby-2.4.1/bin/sass:23:in `load'
+/Users/dickson/.rvm/gems/ruby-2.4.1/bin/sass:23:in `<main>'
+/Users/dickson/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `eval'
+/Users/dickson/.rvm/gems/ruby-2.4.1/bin/ruby_executable_hooks:15:in `<main>'
+*/
```
Comment (laudickson) [inline app.css]: deleting this too. dunno how this got in here

**Pull request merged**